src/test/test_protocol_nmea.cpp

```diff
@@ -0,0 +1,1283 @@
+#include <gtest/gtest.h>
+#include <vector>
+#include "../protocol_nmea.h"
+#include "../ISEarth.h"
+using namespace std;
+
+
+#define ASCII_BUF_LEN   200
+#define POS_LAT_DEG     40.330578
+#define POS_LON_DEG     -111.725816
+#define POS_ALT_M       1406.39
+#define LEAP_SEC        18
+
+TEST(protocol_nmea, nmea_parse_ascb)
```

Comment:
We should also start including a doc block for each of our tests, which describes what the tests do.  Not a requirement, just a suggestion.  It would help all of us to better understand what the individual tests aims to accomplish, and help to determine when it fails, if it failed to catch an edge case, or if there is bigger issue in the test.